app/views/guests/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>Guest Show Page</h1>
+
+<div id="guest_info">
+  <h2>Guest Info</h2>
+  <p><b>NAME: </b><%= @guest.name %></p>
+</div>
+
+<div id="guest_room_history">
+  <h2><%= @guest.name %>'s Room History</h2>
+  
+  <% @guest.rooms.each do |room| %>
+    <div id="room-<%= room.id %>">
+      <h3><%= room.suite %> Suite</h3>
+      <ul>
+        <p><b>RATE: </b>$<%= room.rate %>.00 per night</p>
+        <p><b>HOTEL: </b>$<%= room.hotel_name %></p>
+      </ul>
+    <div>
+  <% end %>
+</div>
+
+<div id="add-room">
+  <h3>Add Room</h3>
+  <p>Enter the ID of an eisting room.</p>
+  <%= form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |f| %>
+    <%= f.label :room_id, "Room ID" %>
+    <%= f.text_field :room_id %>
+    <%= f.submit "Submit" %>
+  <% end %>
+</div>
+
+
+<!-- Idea if I have time: Maybe put in a partial for displaying a hotel? -->
```

Comment:
This might make sense if it's repeated code; but I'm not sure where a hotel is repeated code here...? Either way it might be good if you want to practice it!